src/components/spectrogram/spectrogram.ts

```diff
@@ -510,8 +503,14 @@ export class SpectrogramComponent extends SignalWatcher(ChromeHost(LitElement))
       return;
     }
 
+    // We do not use a truthy assertion here because if the bufferedFrameRef
+    // value is zero, we still want to cancel the animation frame.
+    if (this.bufferedFrameRef !== null) {
+      window.cancelAnimationFrame(this.bufferedFrameRef);
+    }
+
```

Comment:
i'm not following the flow of logic here.

Is the purpose of this to cancel any current resize animations, and then restart them?

also it looks like we're only using one frame of the RAF...? why not just use set timeout?